src/App.js

```diff
@@ -1,16 +1,46 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import './App.css';
 
 const App = () => {
+  const initialCopy = chatMessages.map((chatEntry) => {
+    return { ...chatEntry };
+  });
+
+  const [chatData, setChatData] = useState(initialCopy);
+  const [likeCount, setLikeCount] = useState(0);
+
+  const updateLikes = (chatEntryId) => {
+    const updatedChatData = chatData.map((chatEntry) => {
+      if (chatEntry.id === chatEntryId) {
+        return {
+          ...chatEntry,
+          liked: !chatEntry.liked,
+        };
+      }
+      return chatEntry;
+    });
+    setChatData(updatedChatData);
+    updateLikeCount(updatedChatData);
+  };
+
+  const updateLikeCount = (updatedChatData) => {
+    const count = updatedChatData.reduce(
+      (total, chatEntry) => total + (chatEntry.liked ? 1 : 0),
+      0
+    );
+    setLikeCount(count);
+  };
+
   return (
-    <div id="App">
+    <div className="App">
```

Comment:
👀 The App styles in the CSS file target `#App`, not `.App`, so this should be `id`.